nebula-logger/main/log-management/classes/LogHandler.cls

```diff
@@ -38,10 +38,22 @@ public without sharing class LogHandler {
         this.triggerOperationType = Trigger.operationType;
     }
 
+    /**
+     * @description Returns SObject Type that the handler is responsible for processing
+     * @return      The instance of `SObjectType`
+     */
+    public override SObjectType getSObjectType() {
+        return Schema.Log__c.SObjectType;
+    }
+
     /**
      * @description Runs the trigger handler's logic for the `Log__c` custom object
      */
     public void execute() {
+        if (this.configuration.IsEnabled__c == false) {
```

Comment:
It seems like these checks could be consolidated in the conditionals you have at the beginning of the base class’s “executePostProcessorsFor...” methods so that the subclasses don’t always have to remember to do that check. That probably allows the configuration object to not have to be protected visibility, too

Reply:
The problem is that this guard clause is intended to disable the entire handler's execution, not just the post-processors - I've been meaning to add a way to allow handlers to be temporarily disabled (since you can't disable a trigger in a prod org without deploying), but I think it requires keeping these guard clauses within each handler's `execute()` method